db/post_migrate/20250305233449_populate_type_source_in_reviewable.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+class PopulateTypeSourceInReviewable < ActiveRecord::Migration[7.2]
+  def change
+    # Migrate known reviewables to have a type_source
+    # Copied from db/migrate/20250212045125_add_type_source_to_reviewable.rb
+    known_reviewables = {
+      "chat" => %w[ReviewableChatMessage],
+      "core" => %w[ReviewableFlaggedPost ReviewableQueuedPost ReviewableUser ReviewablePost],
+      "discourse-ai" => %w[ReviewableAiChatMessage ReviewableAiPost],
+      "discourse-akismet" => %w[
+        ReviewableAkismetPost
+        ReviewableAkismetPostVotingComment
+        ReviewableAkismetUser
+      ],
+      "discourse-antivirus" => %w[ReviewableUpload],
+      "discourse-category-experts" => %w[ReviewableCategoryExpertSuggestion],
+      "discourse-post-voting" => %w[ReviewablePostVotingComment],
+    }
+
+    known_reviewables.each do |plugin, types|
+      types.each do |type|
+        Reviewable.where(type: type, type_source: "unknown").update_all(type_source: plugin)
```

Comment:
This should be a regular sql query rather than relying on the model code, we try not to include app code in migrations. Do you know about the DB class yet? If not it would be:

```ruby
DB.exec("UPDATE reviewables SET type_source = :plugin_type_source WHERE type_source = 'unknown' AND type = :type")
```

This is using our mini_sql gem. Also worth updating the other migration to do this